contracts/zero-ex/test/storage_uniqueness_test.ts

```diff
@@ -0,0 +1,36 @@
+import { describe } from '@0x/contracts-test-utils';
+import { readdir, readFile } from 'fs';
+import { basename, resolve } from 'path';
+import { promisify } from 'util';
+
+describe('Storage ID uniqueness test', () => {
```

Comment:
we can delete these now

Reply:
I actually updated the regex to extract the `StorageId` enum used in `getStorage()` but forgot to also update the test case descriptions :upside_down_face:. I would never have noticed if you hadn't said anything.